contracts/src/v0.8/dev/automation/test/MercuryRegistry.t.sol

```diff
@@ -0,0 +1,303 @@
+
+  // Feed: USDC/USD
+  // Date: Wednesday, August 30, 2023 5:05:01 PM
+  // Price: $1.00035464
+  bytes s_august30USDCUSDMercuryReport =
+    hex"0006970c13551e2a390246f5eccb62b9be26848e72026830f4688f49201b5a050000000000000000000000000000000000000000000000000000000001c89843000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000e00000000000000000000000000000000000000000000000000000000000000220000000000000000000000000000000000000000000000000000000000000028000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000120a5b07943b89e2c278fc8a2754e2854316e03cb959f6d323c2d5da218fb6b0ff80000000000000000000000000000000000000000000000000000000064ef69fa0000000000000000000000000000000000000000000000000000000005f5da000000000000000000000000000000000000000000000000000000000005f5b0f80000000000000000000000000000000000000000000000000000000005f5f8b0000000000000000000000000000000000000000000000000000000000240057307d0a0421d25328cb6dcfc5d0e211ff0580baaaf104e9877fc52cf2e8ec0aa7d00000000000000000000000000000000000000000000000000000000024005730000000000000000000000000000000000000000000000000000000064ef69fa0000000000000000000000000000000000000000000000000000000000000002b9e7fb46f1e9d22a1156024dc2bbf2bc6d337e0a2d78aaa3fb6e43b880217e5897732b516e39074ef4dcda488733bfee80c0a10714b94621cd93df6842373cf5000000000000000000000000000000000000000000000000000000000000000205ca5f8da9d6ae01ec6d85c681e536043323405b3b8a15e4d2a288e02dac32f10b2294593e270a4bbf53b0c4978b725293e85e49685f1d3ce915ff670ab6612f";
+
+  function setUp() public virtual {
+    // Set owner, and fork Arbitrum Goerli Testnet (chain ID 421613).
+    // A public Arbitrum Goerli RPC url is being used, and the fork should be cached in CI so availability is not an issue for test runs.
+    vm.startPrank(OWNER);
+    vm.selectFork(vm.createFork("https://goerli-rollup.arbitrum.io/rpc", BLOCK_NUMBER));
```

Comment:
Are these kinds of tests reproducible? I wonder if they might be inherently flaky due to this forking behavior.

Reply:
Ah I just saw the comment on the fork being cached in CI. I wonder how that caching works, will look into it.

Reply:
I think we should figure this out org-wide, as Forks are a very useful part of Foundry testing.